nautobot_ssot/integrations/servicenow/jobs.py

```diff
@@ -20,13 +20,12 @@
 class ServiceNowDataTarget(DataTarget, Job):  # pylint: disable=abstract-method
     """Job syncing data from Nautobot to ServiceNow."""
 
-    debug = BooleanVar(description="Enable for more verbose logging.")
+    debug = BooleanVar(description="Enable for more verbose logging.", default=True)
 
-    # TODO: not yet implemented
-    # delete_records = BooleanVar(
-    #     description="Delete records from ServiceNow if not present in Nautobot",
-    #     default=False,
-    # )
+    delete_records = BooleanVar(
```

Comment:
Is this used anywhere?

Reply:
There are several areas in the models.py file that use diffsync.job.logger.debug but I believe I defaulted the debug value to True in the jobs.py file when I was testing and must have forgotten to set it back to False.

Reply:
I have removed the "default=True" for the delete checkbox in the latest commits.